ts/api/api.ts

```diff
@@ -325,6 +332,23 @@ class OpenYoloApiImpl implements OpenYoloApi {
         this.areTimeoutsDisabled ? undefined : DEFAULT_TIMEOUTS.proxyLogin;
     return request.dispatch(credential, timeoutMs);
   }
+
+  private async cancelLastOperationUsingBrowser() {
+    return Promise.reject('not implemented');
+  }
+
+  async cancelLastOperation() {
```

Comment:
Yes. My next update will make sure i test this via a demo provider like @iainmcgin suggested.